modules/core/example/js/eth/create-wallet-address.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Create a new receive address on an existing multi-sig ethereum wallet at BitGo.
+ *
+ * Copyright 2021, BitGo, Inc.  All Rights Reserved.
+ */
+ const BitGoJS = require('bitgo');
+
+ // change this to env: 'production' when you are ready for production
+ const bitgo = new BitGoJS.BitGo({ env: 'test' });
```

Comment:
Nit: could provide the access token using the `accessToken` constructor param.

Reply:
added typescript examples